### DISCUSSION TITLE
No Volume Control and Choppy Audio on LG HDR 4K

### DISCUSSION THREAD

**Discussion by unteer** [MonitorControl/MonitorControl#938] [Q&A]
I am trying to use Motion Control with my MacbookPro M1 (13-inch 2020). The audio is extremely faint and stutters. Volume control visually adjusts the sliders, but does not actually change the volume.

- OS version 12.2 (21D49)
- Monitor is an LG 32UK550/UL500
- I am connecting to the Monitor through a Dell WD19TB Thunderbolt Dock.
- I am using a DisplayPort connection, and DP 1.2 is enabled on the Monitor.

I compiled and ran the m1dcc command line application. It correctly identifies the monitor:
<img width="458" alt="image" src="https://user-images.githubusercontent.com/6366512/153215993-8419d3fd-b6df-4106-b141-8bfa6bf31f2c.png">

But when I attempt to change the volume, i receive an error: I2C communication failure: (iokit/?) unknown subsystem error
<img width="486" alt="image" src="https://user-images.githubusercontent.com/6366512/153216118-a33c4387-f137-4bd7-8582-40323015adcd.png">

Any thoughts or tips would be appreciated! Thanks!

**Comment by waydabber** [comment]
Do you experience the same when you try using a direct connection and do not use the Thunderbolt dock?
